test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -675,6 +677,184 @@ public void testGetDocIds() throws Exception {
   }
 
   @Test
+  public void testGetDocIdsActionColumnDelete() throws Exception {
```

Comment:
It seems reasonable to me to have four rows in the table instead of four tests, with the benefit of testing to make sure the actions are truly row-based.